src/lib/blueprint.ts

```diff
@@ -117,14 +121,327 @@ interface ObjectProperty extends BaseProperty {
 
 type Method = 'GET' | 'POST' | 'PUT' | 'DELETE' | 'PATCH'
 
+type LowercaseMethod = Lowercase<Method>
+
 export interface TypesModule {
-  openapi: Openapi
+  openapi: OpenAPI
 }
 
 export const createBlueprint = ({ openapi }: TypesModule): Blueprint => {
+  const isFakeData = openapi.info.title === 'Foo'
+  const targetPath = '/acs/systems/list'
+  const targetSchema = 'acs_system'
+
   return {
     name: openapi.info.title,
-    routes: [],
-    resources: {},
+    routes: createRoutes(openapi.paths, isFakeData, targetPath),
+    resources: createResources(
+      openapi.components.schemas,
+      isFakeData,
+      targetSchema,
+    ),
+  }
+}
+
+const createRoutes = (
+  paths: OpenAPIPaths,
+  isFakeData: boolean,
+  targetPath: string,
+): Route[] => {
+  return Object.entries(paths)
+    .filter(([path]) => isFakeData || path === targetPath)
+    .map(([path, pathItem]) => createRoute(path, pathItem))
+}
+
+const createRoute = (path: string, pathItem: OpenAPIPathItem): Route => {
+  const pathParts = path.split('/')
+  const routePath = `/${pathParts.slice(1, -1).join('/')}`
+
+  return {
+    path: routePath,
+    namespace: { path: `/${pathParts[1]}` },
+    endpoints: createEndpoints(path, pathItem),
+    subroutes: [],
+  }
+}
+
+const createEndpoints = (
+  path: string,
+  pathItem: OpenAPIPathItem,
+): Endpoint[] => {
+  return Object.entries(pathItem)
+    .filter(
+      ([, operation]) => typeof operation === 'object' && operation !== null,
+    )
+    .map(([method, operation]) =>
+      createEndpoint(method, operation as OpenAPIOperation, path),
+    )
+}
+
+const createEndpoint = (
+  method: string,
```

Comment:
If there are two kinds of methods let's track them by always calling the openapi version `openapiMethod`.
```suggestion
  openapiMethod: OpenapiMethod,
```